components/Hero.tsx

```diff
@@ -1,53 +1,21 @@
-import React from "react"
-import { Button } from "@/components/ui/button"
-import NavbarAnima from "@/components/header/NavbarAnima"
-import { TypingAnimation } from "@/components/magicui/typing-animation"
+import { ReactNode } from "react"
 
-interface HeroProps {
-  onViewEventsClick: () => void
-}
-
-export const Hero: React.FC<HeroProps> = ({ onViewEventsClick }) => {
+export const Hero = ({
+  image,
+  children,
+}: {
+  image: string
+  children: ReactNode
+}) => {
   return (
     <div
       className="w-full h-[1200px] bg-cover bg-center relative flex flex-col overflow-hidden m-0 p-0"
```

Comment:
I think this `h-[1200px]` is too much?
```suggestion
      className="w-full h-screen bg-cover bg-center relative flex flex-col overflow-hidden m-0 p-0"
```

Reply:
`h-screen` means `height: 100vh` which means 100% of the viewport height

Reply:
So like 1 screen height

Reply:
Thanks! Made that change.